### PR TITLE
fixed #2349 color overiden in zim host activity

### LINKED ISSUE #2349: "Start Server" in ZimHostActivity does not have its color overriden.
**Describe the bug**

The color of the start server button should be green, and the stop server button should be red. Currently, it is a shade of `denim_blue`.
**Expected behavior**

When the button has the text "Start Server" its color should be `startServerGreen`. 
When the button has the text "Stop Server" its color should be `stopServerRed`. 
**Steps to reproduce the behavior:**
1. Open ZimHostActivity by clicking the drawer -> Host Books. 

**Screenshots**
Screenshot_1598790376
Screenshot_1598790372

### PR DESCRIPTION
**Pull request by gouri-panda**
Fixes #2349 

**Changes**
Changed button background to backgroundTint.

### CODE REVIEW

File: app/src/main/res/layout/activity_zim_host.xml [hunk 1]
```diff
@@ -34,13 +34,13 @@
   <Button
     android:id="@+id/startServerButton"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="60dp"
```
Comment (macgills) [inline app/src/main/res/layout/activity_zim_host.xml:37]: why this change?

File: app/src/main/res/layout/activity_zim_host.xml [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
     android:layout_marginStart="8dp"
     android:layout_marginLeft="8dp"
     android:layout_marginEnd="8dp"
     android:layout_marginRight="8dp"
     android:layout_marginBottom="8dp"
-    android:background="@color/startServerGreen"
+    android:backgroundTint="@color/startServerGreen"
```
Comment (macgills) [inline app/src/main/res/layout/activity_zim_host.xml:43]: This also doesn't make sense in light of the original ticket, red wasn't showing up also? Have you tried this in light theme as well?

Comment (gouri-panda) [reply app/src/main/res/layout/activity_zim_host.xml:43]: yes @macgills I updated in the PR.

File: app/src/main/res/layout/activity_zim_host.xml (hunk 1 above)
Comment (gouri-panda) [reply app/src/main/res/layout/activity_zim_host.xml:37]: because in wrap content the height of the button becomes the default.

Comment (gouri-panda) [reply app/src/main/res/layout/activity_zim_host.xml:37]: I just checked in the release version of the app the height of the button is the default. Should I change to wrap_content?

Comment (macgills) [reply app/src/main/res/layout/activity_zim_host.xml:37]: yeah, this ticket wasn't about changing the button height

File: app/src/main/res/layout/activity_zim_host.xml (hunk 2 above)
Comment (macgills) [reply app/src/main/res/layout/activity_zim_host.xml:43]: hmmm @Frans-Lukas can you test this? Seems funky, right? I don't know how your device could have affected it but I want to remove the possibility

File: app/src/main/res/layout/activity_zim_host.xml (hunk 1 above)
Comment (macgills) [reply app/src/main/res/layout/activity_zim_host.xml:37]: Also I should correct the notion of a "default" for height, no such thing exists. There are 4 options `wrap_content`, `match_parent`, the irrationally hated (by me at least) `fill_parent` and specifying a value.
If you don't specify a height then the app will refuse to compile.
Buttons do mandate a `minHeight/Width` in their default styling to comply with the guidelines on touch areas though, if memory serves

Comment (gouri-panda) [reply app/src/main/res/layout/activity_zim_host.xml:37]: Should i remove the height?

Comment (macgills) [reply app/src/main/res/layout/activity_zim_host.xml:37]: revert your change and leave the height as it was, `wrap_content`

File: app/src/main/res/layout/activity_zim_host.xml (hunk 2 above)
Comment (Frans-Lukas) [reply app/src/main/res/layout/activity_zim_host.xml:43]: Hmm, as long as the color updates dynamically. Will check this weekend.

Comment (Frans-Lukas) [reply app/src/main/res/layout/activity_zim_host.xml:43]: @macgills @gouri-panda This works fine for me 👍

**Review by macgills — APPROVED** [review]
